pom.xml

```diff
@@ -1593,7 +1593,7 @@
          comes in via hbase-thirdparty hbase-shaded-netty-->
     <netty.hadoop.version>3.10.5.Final</netty.hadoop.version>
     <!-- end HBASE-15925 default hadoop compatibility values -->
-    <audience-annotations.version>0.5.0</audience-annotations.version>
+    <audience-annotations.version>0.12.0</audience-annotations.version>
```

Comment:
IIRC this was being left alone out of an abundance of caution since it is exposed to our downstream client users.  Since test infra changes need to go to all branches could we break it into a follow on so we can better target specific branches and release note it separately?